modules/setting-up-cpu-manager.adoc

```diff
@@ -9,21 +9,19 @@
 
```

Comment:
There should be some short of introduction to the procedure you're about to follow here. 

https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs